### DISCUSSION TITLE
Quarto website as a knowledge base / note-taking tool?

### DISCUSSION THREAD

**Discussion by paniterka** [quarto-dev/quarto-cli#4509] [Feature Requests]
Hi everyone, 
I have been looking into various knowledge management tools and I was wondering - has anyone tried to use the Quarto blog or book project not as such, but to create an interlinked knowledge base, with backlinks, references etc.? Do you have any examples or experiences to share? 
Thanks in advance for your input!

**Comment by mcanouil** [comment]
I don't know what you mean, but isn't it what <https://quarto.org> is, namely a "knowledge base"?

**Reply by cscheid** [reply]
I think they mean using quarto instead of (eg) [Obsidian](https://obsidian.md/) or other mindmapping applications based on Markdown.

**Reply by mcanouil** [reply]
Oh, I had a quick and furtive thought it was related to Obsidian. 😅
(By definition <https://quarto.org> is a "knowledge base" ^^)

**Reply by paniterka** [reply]
Yes, by "knowledge base" I meant a digital garden or a second brain :) 
https://github.com/MaggieAppleton/digital-gardeners

**Reply by fbob** [reply]
Yep, I vote for a native support of Quarto's .qmd file in the excellent Obsidian interface. 
Tight integration between the obsidian's ecosystem and quarto's power could be indeed very interesting.
They both have very mature strengths and their combination could give something very appealing.
